### ISSUE TITLE
Does this work and is it safe? 

### ISSUE THREAD

**Issue by EnDeRWiGiN** [jtrent238/SteamPortable#1]
Does this work normally or does it have any catches? 

I would love a little more information on what it is doing.

**Comment by jtrent238** [comment]
Yes it basically is just the original steam.exe file that auto-updates itself. This is built with the [PortableApps Platform](https://portableapps.com).

**Comment by THEFLYINGHOBO101** [comment]
well it did start up and it started updating and it dident trigger falcon strike(i installed it on my school computer) and falcon strike is really sensative so yes its safe but it dident install the drivers

**Comment by THEFLYINGHOBO101** [comment]
but it also fixed some of my issues with my games that are totallly not pirated

**Comment by DaddiesCaddie** [comment]
I know this is late but I just got this and it acts as if I am not connected to the internet on my pc. I checked the bootstrap log and this confirms. Is there a way to force the update? Any information on how to get this to work for me would be great! Btw I am online.

**Comment by jtrent238** [comment]
When I get my computer I will see if I can fix it.

**Comment by DaddiesCaddie** [comment]
Is this a type of workaround to actually installing steam? I don't have admin priv on this computer and I have tried the .bat thing but that did not work. I do have a steam account tho just can't download to here.
